test/pylib/runner.py

```
@@ -945,35 +939,14 @@ async def uninstall() -> None:
-        # so the pool doesn't return a broken cluster to tests
+        # so a broken cluster is never handed out to tests
         if cluster.start_exception is not None:
             # Clean up the broken cluster before raising
             try:
-                await cluster.stop()
-                if cluster.api is not None:
-                    cluster.api.close()
-                    cluster.api = None
-                await cluster.release_ips()
+                await cluster.recycle()
             except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)